spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -4615,21 +4615,10 @@ auto configuration supports all HIGH importance properties, some selected MEDIUM
 and any that do not have a default value.
 
 Only a subset of the properties supported by Kafka are available via the `KafkaProperties`
-class. If you wish to configure the producer or consumer with additional properties, you
-can override the producer factory and/or consumer factory bean, adding additional
-properties, for example:
-
-[source,java,indent=0]
-----
-	@Bean
-	public ProducerFactory<?, ?> kafkaProducerFactory(KafkaProperties properties) {
-		Map<String, Object> producerProperties = properties.buildProducerProperties();
-		producerProperties.put("some.property", "some.value");
-		return new DefaultKafkaProducerFactory<Object, Object>(producerProperties);
-	}
```

Comment:
I think this sample should be still here, although re-phrased.
There are some cases which aren't covered with the current `KafkaProperties` , (e.g. `CommonClientConfigs.METRIC_REPORTER_CLASSES_CONFIG`), but I definitely may be interested in different values for `consumer` and for `producer` respectively.